### PR TITLE
Split up readme into Usage, and driveby fix plugins doc

### PR DESCRIPTION
**Pull request by 0atman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,85 +29,7 @@ git clone https://github.com/alphagov/gds-pre-commit.git ~/.gds-pre-commit/
...
-This pre-commit hook has been tested and is working with the following editors:
-
-* Atom
-* VScode
-* IntelliJ
-* Pycharm
-* Emacs
-* Vim (with Fugitive.vim)
-
+ - [Usage](usage.md)
```
Comment (pritchyspritch) [inline README.md:34]: I think the Usage should be in the main README, don't we just want to split off the information that's not critical to set up and running the tool? @0atman @danjoneslf

**Comment by alice-carr** [comment]
@pritchyspritch 's PR #41 has made a few changes to the usage section (and I've made a few comments on that section on that PR) we might want to update that stuff first before splitting it out?

File: README.md (hunk 1 above)
Comment (0atman) [reply README.md:34]: That's not what it says here https://trello.com/c/uzKZhQZC/241-gds-pre-commit-readme-remove-content-after-usage-into-a-separate-linked-md-file

Comment (0atman) [reply README.md:34]: Also, the Usage is WAAAAY to complicated for the normal user, I agree with the decision to split it up.

Comment (pritchyspritch) [reply README.md:34]: Yes it is...Remove content _after_ "Usage"

**Comment by 0atman** [comment]
@alice-carr ah, thanks. I'll merge carefully?

**Comment by 0atman** [comment]
I'd say this is blocked on https://github.com/alphagov/gds-pre-commit/pull/41

File: README.md (hunk 1 above)
Comment (pritchyspritch) [reply README.md:34]: Two sentences with an output snippet for each too complicated for developers?

**Comment by 0atman** [comment]
@alice-carr could you check I've merged your changes correctly :ok_hand:

**Review by alice-carr — APPROVED** [review]

**Pull request merged**